packages/SymbioticController/src/Adapter/Nette/Application/PresenterFactory.php

```diff
@@ -62,8 +62,12 @@ public function createPresenter($name)
      */
     public function getPresenterClass(&$presenterName): string
     {
-        if (isset($this->cache[$presenterName])) {
-            return $this->cache[$presenterName];
+        if (class_exists($presenterName)) {
```

Comment:
Yep, i my tests fails here - basically when any string that is existing non-presenter class will break this.

Maybe this stack trace will help you:
```
1) OdbavTo\Tests\Integration\Api\LoginPresenterTest::invalidPasswordShouldNotPass
Error: Call to undefined method Error::run()

/Users/janmikes/Sites/odbavto-new/vendor/symplify/symbiotic-controller/src/Adapter/Nette/Application/InvokablePresenterAwareApplication.php:180
/Users/janmikes/Sites/odbavto-new/vendor/symplify/symbiotic-controller/src/Adapter/Nette/Application/InvokablePresenterAwareApplication.php:110
/Users/janmikes/Sites/odbavto-new/vendor/nette/application/src/Application/Application.php:179
/Users/janmikes/Sites/odbavto-new/vendor/symplify/symbiotic-controller/src/Adapter/Nette/Application/InvokablePresenterAwareApplication.php:138
/Users/janmikes/Sites/odbavto-new/vendor/symplify/symbiotic-controller/src/Adapter/Nette/Application/InvokablePresenterAwareApplication.php:84
/Users/janmikes/Sites/odbavto-new/vendor/vaclav-sir/n-browser-kit/src/Client.php:96
/Users/janmikes/Sites/odbavto-new/vendor/symfony/browser-kit/Client.php:315
/Users/janmikes/Sites/odbavto-new/tests/Integration/Api/LoginPresenterTest.php:44
```